src/test/java/org/opensearch/sdk/TestExtensionsRunner.java

```diff
@@ -155,15 +153,15 @@ public void testHandleExtensionInitRequest() throws UnknownHostException {
     @Test
     public void testHandleExtensionRestRequest() throws Exception {
 
-        ExtensionTokenProcessor ext = new ExtensionTokenProcessor(EXTENSION_NAME);
```

Comment:
Looks like we are removing all security changes? 
Is it because they aren't on main?

Reply:
Yeah, we can't have that functionality until the feature/identity branch is merged into main

Reply:
Makes sense, could you open an issue. This is something security folks to take care of. 
cc: @peternied @cwperks

Reply:
https://github.com/opensearch-project/opensearch-sdk-java/issues/298

Reply:
Can we please put either put a TODO comment here noting this is a "token placeholder" and pointing to #298, or just rename the variable with a "token placeholder" name?

Reply:
I renamed the variable